handbook/quality/README.md

```diff
@@ -0,0 +1,174 @@
+
+### After reproduced
+After it's in a release formally, the bug should be treated like any other piece of work per the standard engineering process.
+
+### Fast track for Fleeties
+Fleeties do not have to wait for QA to reproduce the bug. If you're confident it's reproducible, it's a bug, and the reproduction steps are well documented, it can be moved directly to the reproduced state.
+
+### Release testing
+When a release is in testing, QA should use the the Slack channel #help-release-bugs to keep everyone aware of issues found. All bugs found should be reported in the channel after creating the bug first.
+
+In the #help-release-bugs channel, product may decide whether the bug is a release blocker. When a release-blocking bug is found, product is responsible for communicating the delay to company stakeholders. 
+
+Release blockers include:
+1. Critical bugs (defined below)
+2. New functionality that impacts previous stable functionality
+3. Incomplete features as defined or implied in the specs
+
+Release blockers must be fixed before a release can be cut. Non-release-blocking bugs may be addressed during a subsequent release per the standard bug process (defined above).
+
+### Critical bugs
+A critical bug is defined as behavior that: 
+* Blocks the normal use a workflow
+* Prevents upgrades to Fleet
+* Causes irreversible damage, such as data loss
+* Introduces a security vulnerability
+
+We need to inform customers and the community about critical bugs immediately so they don’t trigger it themselves. When a bug meeting the definition of critical is found, the bug finder is responsible for raising an alarm.
+Raising an alarm means pinging @here in the #help-product channel with the filed bug.
+
+If the bug finder is not a Fleetie (e.g., a member of the community), then whoever sees the critical bug should raise the alarm. (We would expect this to be customer experience in the community Slack or QA in the bug inbox, though it could be anyone.)
+Note that the bug finder here is NOT necessarily the **first** person who sees the bug. If you come across a bug you think is critical, but it has not been escalated, raise the alarm!
+
+Once raised, product confirms whether or not it's critical and defines expected behavior.
+When outside of working hours for the product team or if no one from product responds within 1 hour, then fall back to the #help-p1.
+
+Once the critical bug is confirmed, customer experience needs to ping both customers and the community to warn them. If CX is not available, the oncall engineer is responsible for doing this.
+If a quick fix workaround exists, that should be communicated as well for those who are already upgraded.
+
+When a critical bug is identified, we will then follow the patch release process in [our documentation](https://fleetdm.com/docs/contributing/releasing-fleet#patch-releases).
+
+### Measurement
```

Comment:
Hey @CharlotteChance, this is the new set of bug metrics for the weekly update we had discussed. If you go below, you can click the various filters to get the number. WDYT?

Reply:
Hey, @zhumo, thanks for the tag. 
Here's where I see some crossover between what we have in the weekly update and what is listed in the PR: 

"Bugs reported", and "Bugs reported in released code"(this one is new and will need a link in the spreadsheet) == Bugs opened this week, and Total # of bugs open

- "bugs reported" and "bugs opened this week" seem like the same thing, and since this is a weekly report, I think we can safely consolidate the two 
- "bugs reported in released code" is new so we'll need to sync up on what is meant and add the reference link to the spreadsheet to call this done
- "total # of bugs open" is what "bugs reported" records

"Bugs closed" == # ^of^ bugs closed this week 

- since this is a weekly update these seem like the same thing, and I think we're needing to sync up on the wording

Are you looking to rename these columns and keep the metrics or is the idea to completely scrap the current columns and replace them with the PR suggestions?

Reply:
Ah ok great. I don't feel strongly about the naming, though I would like the weekly update spreadsheet and this handbook doc to be aligned. Your proposals about merging ones that seem the same make sense! I will schedule a call to make sure we're on the same page about all these. I also want to make sure how you're counting is how I'm counting.

Reply:
@zhumo Sounds good, see you soon!